actions/data/event.ts

```diff
@@ -50,3 +58,87 @@ export async function getEventByUrl(url: string) {
     return null;
   }
 }
+
```

Comment:
Make thi snto one update

Reply:
we gonna have to leave this one for a next time :(